autogpt_platform/frontend/src/components/__legacy__/Wallet.tsx

```diff
@@ -193,6 +194,42 @@ export default function Wallet() {
     setCompletedCount(completed);
   }, [groups, state?.completedSteps]);
 
+  // Load last seen credits from localStorage once on mount
+  useEffect(() => {
+    try {
+      const stored = localStorage.getItem("wallet_last_seen_credits");
```

Comment:
When is this `wallet_last_seen_credits` being set?  what happened when it's being set since we only does this once